.gitignore

```diff
@@ -37,6 +37,7 @@ vtest/META-INF
 vtest/Thumbnails
 vtest/Pictures
 .vs
+.vscode
```

Comment:
evidently no one else uses vscode, and I just switched to it, so this line should be OK to add. We have .vs already, after all.